### DISCUSSION TITLE
Too small project folder font size

### DISCUSSION THREAD

**Discussion by ryota-murakami** [zed-industries/zed#6610] [Feature Requests]
How can I setting that font size?
![Screenshot 2023-10-18 at 15 09 19](https://github.com/zed-industries/community/assets/5501268/e99fbf68-e6f6-448c-a42b-2dd968e32390)

**Comment by widersky** [comment]
For me the font size is too big 😅 and whole files panel is very unreadable for me too. AFAIK there is no option to change font size yet.

**Reply by ryota-murakami** [reply]
@widersky Yeah there are font size option for main editor and terminal, but missing Project Panel 😭
https://docs.zed.dev/configuration/configuring-zed#project-panel

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
